src/emcpy/plots/skewt_projection.py

```diff
@@ -0,0 +1,125 @@
+from contextlib import ExitStack
+
+from matplotlib.axes import Axes
+import matplotlib.transforms as transforms
+import matplotlib.axis as maxis
+import matplotlib.spines as mspines
+
+
+# The sole purpose of this class is to look at the upper, lower, or total
```

Comment:
was most of this "borrowed" from the matplotlib examples? if so, perhaps add a reference somewhere?

Reply:
All of it was borrowed from here: https://matplotlib.org/stable/gallery/specialty_plots/skewt.html. MetPy did the same, but I will definitely add a reference.